jobs/MergeFreezer/merge_freezer.groovy

```diff
@@ -3,6 +3,7 @@ node {
     checkout scm
     stage("Launch MergeFreezer"){
         withCredentials([string(credentialsId: 'JENKINSRHD_GITHUB_TOKEN', variable: 'GITHUB_TOKEN'),
+                         string(credentialsId: 'PULLER_GITHUB_TOKEN_POOL', variable: 'PULLER_GITHUB_TOKEN_POOL'),
```

Comment:
what kind of secure text is the PULLER_GITHUB_TOKEN_POOL  in jenkins? please share an example